README.md

```diff
@@ -7,15 +7,9 @@
 [![GitHub Issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-library/module/docusign.dsesign.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-library/labels/module%2Fdocusign.dsesign)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-DocuSign eSignature is a document management and electronic signature service developed by DocuSign. It allows users to securely sign, send, and manage documents in a digital environment. This connector provides the capability to programmatically perform DocuSign eSignature related operations. For more information about configuration and operations, refer to the module.
+DocuSign is a digital transaction management platform that enables users to securely sign, send, and manage documents electronically.
```

Comment:
Shall we link this to official DocuSign website?

Reply:
Fixed in [067eb61](https://github.com/ballerina-platform/module-ballerinax-docusign.dsesign/pull/18/commits/067eb61125d726a141a4b4d28d203e6720a9837a)